api/sign_transaction.go

```diff
@@ -10,9 +10,52 @@ import (
 	"github.com/onflow/flow-go/fvm/evm/emulator"
 )
 
+const defaultGasLimit uint64 = 15_000_000
+
 var key, _ = gethCrypto.HexToECDSA("45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8")
 var signer = emulator.GetDefaultSigner()
 
+// signTxFromArgs will create a transaction from the given arguments and sign it
+// with a test account. The resulting signed transaction is only supposed to be
+// used through `EVM.run` inside Cadence scripts, meaning that no state change
+// will occur. This is only useful for `eth_estimateGas` and `eth_call` endpoints.
+func signTxFromArgs(args TransactionArgs) ([]byte, error) {
```

Comment:
In a follow-up PR, I will remove `signGasEstimationTx` and use the newly-added `signTxFromArgs`

Reply:
maybe just add a todo so we don't forget

Reply:
It is next up on my list, I will tackle it as part of https://github.com/onflow/flow-evm-gateway/issues/161